apps/aecore/src/aec_tx_sign.erl

```diff
@@ -147,11 +148,30 @@ serialize_for_client(Encoding, Header, #signed_tx{}=S) ->
 
 serialize_for_client(message_pack, #signed_tx{}=S, BlockHeight, BlockHash) ->
     GenericData = [#{<<"block_height">> => BlockHeight},
-                   #{<<"block_hash">> => BlockHash}],
-    aec_base58c:encode(transaction, msgpack:pack(serialize(S) ++ GenericData));
+                   #{<<"block_hash">> => aec_base58c:encode(block_hash, BlockHash)}],
+    aec_base58c:encode(transaction, msgpack:pack(serialize(S) ++ [GenericData]));
```

Comment:
/me trying to understand why putting list of two items inside a list - resulting in list of one item that is list of two items.

Reply:
This is used here https://github.com/aeternity/epoch/pull/644/commits/d85817d364f9f94d03a52ac81ae6cd3489c74308#diff-43772af989581a69a6ac311448a78975R162